src/lib/Microsoft.Health.Events/Model/Event.cs

```diff
@@ -0,0 +1,47 @@
+﻿// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+
+namespace Microsoft.Health.Events.Model
+{
+    public class Event
```

Comment:
Yeah the main reason for creating this "Event" object was not to be tied to event hub dependencies elsewhere in our code.

It isn't really an improvement (other than removing the dependency) since we are just using a similar object instead of the event hub one. I will try and improve this via an interface or something else

Reply:
updated to EventMessage. Not a huge fan but couldn't think of a better name that wasn't already taken